config/query/boosting.yml

```diff
@@ -36,6 +36,8 @@ base:
     true: 0.5
   topical_events:
     coronavirus-covid-19-uk-government-response: 5
+  part_of_taxonomy_tree:
+    5b7b9532-a775-4bd2-a3aa-6ce380184b6c: 5 # /coronavirus-taxon
```

Comment:
This means things tagged to both will be double-boosted

Reply:
But I don't think we have a way around that easily

Reply:
I had just about worked that out, and was trying to decide whether that was a problem 🤔

Reply:
They're close enough that I can retire the topical event one I think.

Reply:
I've now dropped the topical event boost